sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/aio/_text_analytics_client_async.py

```diff
@@ -459,6 +465,8 @@ async def analyze_sentiment(  # type: ignore
             be used for scoring, e.g. "latest", "2019-10-01". If a model-version
             is not specified, the API will default to the latest, non-preview version.
         :keyword bool show_stats: If set to true, response will contain document level statistics.
+        .. versionadded:: v3.1-preview.1
+            The *show_opinion_mining* parameter.
```

Comment:
just curious if this is correct placement or if it should be under the keyword for show_opinion_mining?

Reply:
If i put it under the definition for `show_opinion_mining`, it's splitting up the keyword definition now in a very weird way. Speaking of.. @scbedd do you have any updates for this? no rush if not

Reply:
what if we move show_opinion_mining as the last keyword in docstring? does that help at all?